racket-describe.el

```diff
@@ -25,6 +25,9 @@
 (require 'racket-scribble)
 (require 'racket-browse-url)
 ;; Don't (require 'racket-repl). Mutual dependency. Instead:
+(declare-function racket-file-name-back-to-front "racket-back-end" (file))
```

Comment:
I wonder if the better solution here instead would be to add `(require 'racket-back-end)` above.

Without native-comp, the `(require 'racket-cmd)`, which in turn does `(require 'racket-back-end)`, suffices --- the transitive require means something.

Whereas with native-comp (if I'm guessing correctly?) it wants to see `'racket-back-end` required directly, here?

Reply:
I think you are right. native-comp seems to want direct imports and will fail if the import of a form is transitive.